titanic/titanicML.ipynb

```diff
@@ -414,8 +419,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Next split up the data with the 'train test split' method in the Cross Validation module\n",
```

Comment:
Removed the reference to the cross validation module since scikit-learn's `train_test_split` method is now imported from `sklearn.model_selection`.